Dockerfile-Server

```diff
@@ -0,0 +1,49 @@
+FROM centos:7
```

Comment:
In general, should this mimic how an EC2 is built out? 
If so -- sorry if this blows up lots of things -- but I really liked the idea of using Packer with a Docker builder, and allowing us to still employ our EC2-ansible provisioners with that docker builder. I think there were some comments to that effect in this ticket: (~~can't seem to find it, thought it was under OPS-200~~) this comment:  https://federal-spending-transparency.atlassian.net/browse/OPS-254?focusedCommentId=34453

Perhaps we'd need to refactor our ansible provisioners so we could isolate _just_ the layers needed in this build? If it's not that easy to separate all that would go into an EC2 and the subset that would go into here, maybe just keep them split ... but that does allow drift between how we do it in an EC2 and how we do it in this container. 

But ... that whole concept may lead the goal of this astray. If the intent is to make this _very close_ to the other `Dockerfile` we have, can we get better reuse out of 1) creating a base image the two share? 2) or using the non-Server file _as_ the base image for the server 1, and just adding the extra dependencies, 3) or is there a way with docker multi-stage builds to pull in another docker file as the base series of commands to run (i.e. they both share an included dockerfile?).

Reply:
the ultimate goal of this image is to be able to run our various management commands in an isolated docker environment. Not necessarily run the application. the main docker image built with the regular `Dockerfile` would be responsible for that. Based on _how_ we currently run our management commands is where we get all these additional dependencies (example: pulling a file from S3 using the CLI before running a job)

Reply:
I think it does make sense though to start with the existing image built using `Dockerfile` and then just add stuff for the purposes of running all our management commands in the way we _currently_ run them (don't want to out scope this ticket to refactoring how we run all our jobs)